tgui/packages/tgui/interfaces/SpiderOS.js

```diff
@@ -0,0 +1,1089 @@
+                tooltipPosition="top-left" />
+            </LabeledList.Item>
+            {if_concentration}
+          </LabeledList>
+        </NoticeBox>
+        <NoticeBox className={"NoticeBox_" + color_choice} success={0} danger={0} mt={-1.2} align="center">
+          <Button
+            width="80%"
+            icon="power-off"
+            mt={0.5}
+            textAlign="center"
+            content={dynamicButtonText}
+            tooltip={"Позволяет вам включить костюм и получить доступ к применению всех функций в нём заложенных. \
+              \nУчтите, что вы не сможете приобрести любые модули, когда костюм будет активирован. \
+              \nТак же включённый костюм пассивно потребляет заряд для поддержания работы всех функций и модулей.\
+              \nАктивированный костюм нельзя снять обычным способом, пока он не будет деактивирован.\
+              \nВключение ровно как и выключение костюма занимает много времени. Подумайте дважды прежде, чем выключать его на территории врага!"}
+            tooltipPosition="top-left"
+            onClick={() => act("initialise_suit")} />
+        </NoticeBox>
+      </Flex.Item>
+    </Flex>
+  );
+};
+
+const Helpers = (properties, context) => {
+  const { data } = useBackend((properties, context));
+  const {
+    allActionsPreview,
+  } = data;
+  return (
+    <Section title="Советы и подсказки"
+      style={{ "text-align": "center" }}
+      buttons={<Button
+        content="?"
+        tooltip={"Молодым убийцам часто не легко освоится в полевых условиях, даже после интенсивных тренировок. \
+        \nЭтот раздел призван помочь вам советами по определённым часто возникающим вопросам касательно возможных миссий которые вам выдадут\
+        или рассказать о малоизвестной информации которую вы можете обернуть в свою пользу."}
+        tooltipPosition="bottom-left" />}>
+      <Flex
+        direction="column"
+        grow={1}
+        alignContent="center">
+        <Flex.Item direction="row">
+          <Button
+            className="Button_green"
+            height="48px"
+            width="48px">
+            <img
+              height="48px"
+              width="48px"
+              src={`data:image/jpeg;base64,${allActionsPreview["ninja_teleport"]}`}
+              style={{
+                "margin-left": "-6px",
+                "-ms-interpolation-mode": "nearest-neighbor",
+              }} />
+            <Tooltip
+              title={"Телепортация и шаттл"}
+              content={"В вашем Додзё есть личные устройства для телепортации на обьект вашей миссии. \
+              Точка назначения случайная, но приоритет идёт на технические тоннели станции или малопосещаемые места. \
+              \nЭто отличный способ быстро приступить к выполнению задания. \
+              \nПользуясь встроенным контроллером шатла, вы всегда сможете призвать его к себе и вернуться назад. \
```

Comment:
Наверное лучше сейчас это поменять, а то никто потом не исправит.
```suggestion
              \nПользуясь встроенным контроллером шаттла, вы всегда сможете призвать его к себе и вернуться назад. \
```